en/releases/1.12.md

```diff
@@ -5,26 +5,65 @@
 
 ## Changes
 
+* [Common](#common)
+* [MAVLink](#mavlink)
+* [Commander](#commander)
 * [Multicoper](#multicopter)
+* [VTOL](#vtol)
+* [Control & Navigation](#control)
+
+### Common
+
+* **Remaining Flight Range based RTL Trigger ([PR#16399](https://github.com/PX4/PX4-Autopilot/pull/16399))**
+  * Calculates time to home, on RTL, taking into account vehicle speed, wind speed, and desination distance/direction
+* Pre-emptive geofence breach ([PR#16400](https://github.com/PX4/PX4-Autopilot/pull/16400))
+  * Triggers a breach if the _predicted_ current trajectory will result in a breach, allowing the vehicle to be re-routed to a safe hold position.
+
+### MAVLink
+
+* **MAVLink Ethernet Configuration ([PR#14460](https://github.com/PX4/PX4-Autopilot/pull/14460))**
+  * MAVLink Ethernet channel settings such as UDP port, remote port and broadcast mode now can be changed dynamically via parameters.
+* **Support for COMPONENT_INFORMATION ([PR#16039](https://github.com/PX4/PX4-Autopilot/pull/16039))**
+  * This new message enables a GCS to understand which commands are supported by any given Autopilot (RFC: [mavlink#1339](https://github.com/mavlink/mavlink/issues/1339))
+
+### Commander
+
+* **Commander: use control mode flags and cleanup arm_disarm ([PR#16266](https://github.com/PX4/PX4-Autopilot/pull/16266))**
+  * Consolidates arming logic, and does a better job at keeping the current state
+* **Commander: Separete out manual control setpoint processing ([PR#16878](https://github.com/PX4/PX4-Autopilot/pull/16878))**
+  * Adds a new class `ManualControl` for handling `manual_control_setpoint` and handles RC loss, RC override, and RC arming/disarming
 
 ### Multicopter
 
-* #### Intuitive stick feel in Position mode
+* **More intuitive stick feel in Position mode**
```

Comment:
This needs to go into docs in Position Mode here: http://docs.px4.io/master/en/flight_modes/position_mc.html and possibly also here: http://docs.px4.io/master/en/getting_started/flight_modes.html#position-mode-mc.

I do not understand what these things mean. That's fine here, but they would need to be explained in above docs. For example 
- "what is shunting?", 
- what are these unexpected tilt changes - you mean it is going along at full speed and the tilt of the vehicle - oscillates or something?
- "Horizontal stick input mapped to acceleration instead of velocity setpoints" is just words. Is there some way of explaining why this is more intuitive or do we need to take it on faith?

Reply:
Good catch, I created an issue to track this https://github.com/PX4/PX4-user_guide/issues/1133